featurebyte/api/catalog.py

```diff
@@ -294,6 +346,29 @@ def update_name(self, name: str) -> None:
         """
         self.update(update_payload={"name": name}, allow_update_local=True)
 
+    @typechecked
+    def update_online_store(self, online_store_name: str) -> None:
```

Comment:
Based on my understanding, we don't allow user to remove online store association with the catalog, right?

Reply:
Maybe we should. I can change the type to allow `None`

Reply:
If we want to allow uer to remove online store association, we need to add some special handling as we will exclude `None` when update the document. https://github.com/featurebyte/featurebyte/blob/main/featurebyte/service/base_document.py#L1017